appveyor.yml

```diff
@@ -7,7 +7,7 @@ branches:
     - master
 
 environment:
-  nodejs_version: "6"
+  nodejs_version: "10"
```

Comment:
For a future PR: migrate the Windows CI to CircleCI

For this PR: the CircleCI config needs to use Node 10 too.